slackblocks/blocks.py

```diff
@@ -251,7 +251,7 @@ class ImageBlock(Block):
     def __init__(
         self,
         image_url: str,
-        alt_text: Optional[str] = " ",
```

Comment:
While I get your point, it's not actually an optional argument (i.e. `None` is not a valid value, which is what `Optional` means in the typechecker). If you revert this particular change it should still behave as you intend, without being incorrect from a typing perspective.

Reply:
Ok, I've removed that particular change.